src/analysis/special_functions/exp.lean

```diff
@@ -131,6 +131,9 @@ namespace real
 
 variables {x y z : ℝ}
 
+lemma exp_half (x : ℝ) : exp (x / 2) = sqrt (exp x) :=
```

Comment:
Is `half` the name for `1/2` or `x/2`? I'd maybe lean towards `exp_div_two`, but I don't know the convention in this bit of the library.

Reply:
We have `add_halves`, `half_pos`, and `one_half_pos`.